be1-go/hub/organizer.go

```diff
@@ -54,12 +90,34 @@ func (o *organizerHub) handleIncomingMessage(incomingMessage *IncomingMessage) {
 	log.Printf("organizerHub::handleIncomingMessage: %s", incomingMessage.Message)
 
 	client := incomingMessage.Client
+	byteMessage := incomingMessage.Message
 
-	// unmarshal the message
+	// Check if the GenericMessage has a field "id"
 	genericMsg := &message.GenericMessage{}
-	err := json.Unmarshal(incomingMessage.Message, genericMsg)
+	id, ok := genericMsg.GetID(byteMessage)
```

Comment:
Minor comment: would it make sense to rename `GetID` to `UnmarshallID`? The method name confused me, made me think that the ID wasn't unmarshalled anymore.

Reply:
I totally agree, it would make more sense. I just updated it.